apps/onlyoffice.sh

```diff
@@ -23,6 +23,51 @@ root_check
 # Nextcloud 13 is required.
 lowest_compatible_nc 13
 
+## Onlyoffice for NC 18.0.1 onwwards
```

Comment:
This comes waay to early in the script. 

Need to come after all the checks.

Also, have you tested that this really works?

Nice job btw!

Reply:
this is the "headline" for the section below. (check first if these steps apply and install it the normal way, if not)
And yes, I have tested it and it works.
Thanks!